.travis.yml

```diff
@@ -69,6 +69,14 @@ matrix:
       env: TOXENV=py34
     - python: "3.5"
       env: TOXENV=py35
+    - os: osx
+      osx_image: xcode7.3
```

Comment:
Ah. That makes sense. I think it's fine to leave this as is as long as we add a comment saying something like `cryptography no longer supports OpenSSL 0.9.8`.